flixel/util/FlxPath.hx

```diff
@@ -115,16 +115,47 @@ class FlxPath implements IFlxDestroyable
 	private var _firstUpdate:Bool = false;
 	
 	/**
+	 * An actual array, which holds all the path points.
+	 */
+	private var _nodes:Array<FlxPoint>;
+	
+	/**
 	 * Object which will follow this path
 	 */
 	@:allow(flixel.FlxObject)
 	private var object:FlxObject;
 	
 	/**
 	 * Creates a new FlxPath.
+	 * Every FlxObject have `path` property which can make it move along specified array of way points.
+	 * Everything you need is:
+	 * a) create path
+	 * var path = new FlxPath();
+	 * b) specify array of way points
+	 * var points:Array<FlxPoint> = [new FlxPoint(0, 0), new FlxPoint(100, 0)];
+	 * c) assign path to object
+	 * object.path = path;
+	 * d) start path
+	 * path.start(points, 50, FlxPath.FORWARD);
+	 * 
+	 * You can do this in one line:
+	 * object.path = new FlxPath().start([new FlxPoint(0, 0), new FlxPoint(100, 0)], 50, FlxPath.FORWARD);
+	 * 
+	 * or using some more chaining:
+	 * object.path = new FlxPath().add(0, 0).add(100, 0).start(50, FlxPath.FORWARD);
+	 * 
+	 * @param	Nodes	An optional array of points which will be used for path.
 	 */
-	public function new() {}
+	public function new(?Nodes:Array<FlxPoint>) 
```

Comment:
What's the benefit of having a `Nodes` param here? You're gonna need to call `start()` anyway if you want to use the path.